README.md

```diff
@@ -49,6 +49,16 @@ arguments list, and run it again. This will fill the database with some initial,
 1. Run the project via `pipenv run python botlistbot/main.py`
 </details>  
 
+<details>
+<summary>Setup a PostgreSQL database instance with Docker</summary>
+
+1. Install Docker
+1. Run `docker-compose up -d`
+1. Set the `DATABASE_URL` variable in the `.env` file to `postgres://botlistbot:botlistbot@localhost:5432/botlistbot`
```

Comment:
If this goes in template.env I think we're good

Reply:
- [ ] Add to template
- [ ] Remove this step

Reply:
Done! :+1: